pkgs/development/python-modules/certifi/python2.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, python3
+}:
+
+let
+  inherit (python3.pkgs) certifi;
+
+in buildPythonPackage rec {
+  pname = "certifi";
+  inherit (certifi) version;
+
+  src = fetchPypi {
+    version = "2019.11.28";
+    inherit pname;
```

Please use the original version:

```suggestion
  version = "2019.11.28";

  src = fetchPypi {
    inherit pname version;
```